### PR TITLE
The usage and manpage for docker instance management command, mkdocke…

### PR DESCRIPTION
**Pull request by zet809**
The usage and manpage for *docker command and rpower start/stop/restart/pause/unpause/state.

### CODE REVIEW

File: docs/source/guides/admin-guides/references/man1/lsdocker.1.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+********
+SYNOPSIS
+********
+
+
+\ **lsdocker**\  \ *noderange*\  [\ **-l**\ |\ **--logs**\ ]
+
+\ **lsdocker**\  \ *dockerhost*\ 
+
+\ **lsdocker**\  [\ **-h**\ |\ **--help**\ ]
```
Comment (junxiawang) [inline docs/source/guides/admin-guides/references/man1/lsdocker.1.rst]: *docker -v and -h could not work please delete this part if -v and -help could not work

File: docs/source/guides/admin-guides/references/man1/mkdocker.1.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+********
+EXAMPLES
+********
+
+
+
+.. code-block:: perl
+
+     mkdocker host01c01 image=ubuntu command=/bin/bash dockerflag="{\"AttachStdin\":true,\"AttachStdout\":true,\"AttachStderr\":true,\"OpenStdin\":true}"
```
Comment (junxiawang) [inline docs/source/guides/admin-guides/references/man1/mkdocker.1.rst]: Please add more examples about how to use mkdocker for the dockerflag is much more flexibler for user.

File: docs/source/guides/admin-guides/references/man1/lsdocker.1.rst (hunk 1 above)
Comment (zet809) [inline docs/source/guides/admin-guides/references/man1/lsdocker.1.rst]: The -v and -h option will work the build that after this pull request is merged.

File: docs/source/guides/admin-guides/references/man1/mkdocker.1.rst (hunk 2 above)
Comment (zet809) [inline docs/source/guides/admin-guides/references/man1/mkdocker.1.rst]: I have added some docker flags can be used when creating docker container. Pls verify.

**Pull request merged**